keychain-sdk/example_keychain_test.go

```diff
@@ -30,9 +30,11 @@ func Main() {
 		Mnemonic:   "virus boat radio apple pilot ask vault exhaust again state doll stereo slide exhibit scissors miss attack boat budget egg bird mask more trick",
 
 		// setup throughput for batching responses
-		GasLimit:      400000,
-		BatchInterval: 8 * time.Second,
-		BatchSize:     10,
+		GasLimit:            400000,
+		AutoEstimateGas:     false,
+		GasAdjustmentFactor: 1.2,
```

Comment:
let's put realistic values with AutoEstimateGas=true in the example, as it'll be the recommended way of doing this